apis/swagger.yml

```diff
@@ -1215,6 +1215,10 @@ definitions:
           Address / URL of the index server that is used for image search,
           and as a default for user authentication.
         type: "string"
+      DefaultRegistry:
```

Comment:
Thanks a lot for add this field `DefaultRegistry ` into system info.

I am wondering if we have put this in the info API's response? @Ace-Tang

Reply:
the filed is not added by me, already included in the system info.